### PR TITLE
Add I18N support for diagnosis.

### PR DESCRIPTION
**Pull request by reafans**
Add I18N support for `Title `of diagnosis table：

17V7BIRVFX

### CODE REVIEW

File: ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx [hunk 1]
```diff
@@ -71,6 +71,8 @@ type Props = {
 
 export default function DiagnosisTable({ diagnosis }: Props) {
   const { Category, Title, CommentEN, Column, Rows } = diagnosis
+  const { t } = useTranslation()
+  const title = 'diagnosis.tables' + Title
```
Comment (baurine) [inline ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx:75]: Miss a dot here: 'diagnosis.tables.'

Comment (reafans) [reply ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx]: ok，done.

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 2]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    tidb/pd_goroutines_count: TiDB/PD Goroutines 数量
+    tikv_thread_cpu_usage: TiKV CPU 使用情况
+    store_status: 存储状态
+    cluster_status: 集群状态
+    etcd_status: Etcd 状态
+    cluster_info: 集群信息
+    cache_hit: 缓存命中率
+    cluster_hardware: RocksDB时间消耗
+    rocksdb_time_consume: RocksDB Time Consume
+    top_10_slow_query: 十大慢查询
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:53]:
```suggestion
    top_10_slow_query: Top10 慢查询
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 3]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    top_10_slow_query_group_by_digest: 按摘要排序的十大慢查询
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:54]:
```suggestion
    top_10_slow_query_group_by_digest: 按 SQL 指纹聚合的 TOP10 慢查询
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 4]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    slow_query_with_diff_plan: 有 Diff 计划的慢查询
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:55]:
```suggestion
    slow_query_with_diff_plan: 有不同执行计划的慢查询
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 5]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    process_cpu_usage: CPU 消耗率
+    process_memory_usage: 内存消耗率
+    tidb/pd_goroutines_count: TiDB/PD Goroutines 数量
+    tikv_thread_cpu_usage: TiKV CPU 使用情况
+    store_status: 存储状态
+    cluster_status: 集群状态
+    etcd_status: Etcd 状态
+    cluster_info: 集群信息
+    cache_hit: 缓存命中率
+    cluster_hardware: RocksDB时间消耗
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:51]:
```suggestion
    cluster_hardware: RocksDB 时间消耗
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 6]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    tikv_error: TiKV 错误
+    tidb_current_config: TiDB当前配置
+    pd_current_config: PD当前配置
+    tikv_current_config: TiKV当前配置
+    node_load_info: 节点负载信息
+    process_cpu_usage: CPU 消耗率
+    process_memory_usage: 内存消耗率
+    tidb/pd_goroutines_count: TiDB/PD Goroutines 数量
+    tikv_thread_cpu_usage: TiKV CPU 使用情况
+    store_status: 存储状态
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:46]:
```suggestion
    store_status: TiKV 节点的存储状态
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 7]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    task_info: 任务信息
+    snapshot_info: 快照信息
+    coprocessor_info: Coprocessor 信息
+    raft_info: Raft 信息
+    tikv_error: TiKV 错误
+    tidb_current_config: TiDB当前配置
+    pd_current_config: PD当前配置
+    tikv_current_config: TiKV当前配置
+    node_load_info: 节点负载信息
+    process_cpu_usage: CPU 消耗率
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:42]:
```suggestion
    process_cpu_usage: 进程 CPU 使用率
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 8]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
+    process_memory_usage: 内存消耗率
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:43]:
```suggestion
    process_memory_usage: 进程的内存消耗率
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml (hunk 7 above)
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:42]:
```suggestion
    process_cpu_usage: 进程的 CPU 使用率
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml (hunk 8 above)
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:43]:
```suggestion
    process_memory_usage: 进程使用的内存
```

File: ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml [hunk 9]
```diff
@@ -4,3 +4,52 @@ diagnosis:
...
   fold: 收起
+  tables:
+    generate_report_error: 生成报告的错误
+    report_time_range: 报告时间范围
+    diagnose: 诊断
+    total_time_consume: 总的时间消耗
+    total_error: 总的错误
+    time_consume: 时间消耗
+    transaction: 事务
+    tidb_connection_count: TiDB连接数
```
Comment (crazycs520) [inline ui/src/externalEntries/diagnoseReport/translations/zh-CN.yaml:15]: 英文后面留一个空格

File: ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx [hunk 10]
```diff
@@ -79,7 +81,7 @@ export default function DiagnosisTable({ diagnosis }: Props) {
           {c}
         </h1>
       ))}
-      <h3 className="is-size-4">{Title}</h3>
+      <h3 className="is-size-4">{t(title)}</h3>
```
Comment (breeswish) [inline ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx:84]:
```
t(`diagnosis.tables.${Title}`)
```

Comment (reafans) [reply ui/src/externalEntries/diagnoseReport/components/DiagnosisTable.tsx:84]: Got it, i will do this change for Title in next pr which would add I18N support for `Category/Comment` of diagnosis table.

**Pull request merged**